src/dropdown-arrow-down.js

```diff
@@ -0,0 +1,13 @@
+import { createElement, Component } from 'preact' /** @jsx createElement */
+
+export default class DropdownArrowDown extends Component {
+  render () {
+    return (
+      <svg version='1.1' xmlns='http://www.w3.org/2000/svg' class='dropdown-arrow-down'>
```

Comment:
Note that the `className` here includes positioning / layout styling:

```css
position: absolute;
right: 8px;
width: 24px;
height: 24px;
top: 10px;
```

This component would be more reusable if the layout styling was specified from its parent, so instead:

```js
<svg version='1.1' xmlns='http://www.w3.org/2000/svg' className={ this.props.className }>
```

And when you call it:

```js
<DropdownArrowDown className={ dropdownArrowClassName } />
```